docs/pt/docs/tutorial/path-params.md

```diff
@@ -0,0 +1,246 @@
+# Parâmetros de rota
+
+Você pode declarar "parâmetros" ou "variáveis" de rota com a mesma sintaxe usada para formatar strings em Python:
```

Comment:
Is there a reason for `parametros` and `variaveis` being wrapped by the double quotes?

Reply:
In English it's the same. https://fastapi.tiangolo.com/tutorial/path-params/#path-parameters

Reply:
Ah... ok.